src/python/doc/representations.rst

```diff
@@ -80,3 +133,11 @@ Metrics
    :members:
    :special-members:
    :show-inheritance:
+
+PersLay
+-------
+.. automodule:: gudhi.tensorflow.perslay
```

Comment:
Note that this way we end up documenting it as `gudhi.tensorflow.perslay.Perslay`. If you want to document it as `gudhi.tensorflow.Perslay`, we have to do it slightly differently I guess.

Reply:
Thanks, I've updated it and it works now.